.github/workflows/full-ci.yml

```diff
@@ -0,0 +1,62 @@
+name: full-ci
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  run-tests:
+    uses: ./.github/workflows/_tests.yml
+
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    needs: [run-tests]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: pdm-project/setup-pdm@main
+        name: Setup PDM
+        with:
+          python-version: 3.8
+          architecture: x64
+          enable-pep582: true
+      - name: Add PDM Publish plugin
+        run: pdm plugin add pdm-publish
```

Comment:
I think you don't have to install `pdm-publish`. These commands are already included in pdm.

Reply:
True, skipped this step but had to expand step below